scripts/build_pkgdown.R

```diff
@@ -50,4 +80,36 @@ for (pkg in packages) {
   })
 }
 
+logger("Creating index page")
+
+built_pkg_dirs <- list.dirs(output_dir, recursive=FALSE, full.names = FALSE)
+before_text <- c(
+  '<!DOCTYPE html>',
+  '<html lang="en">',
+  '<head>',
+  '  <title>Package-specific documentation for the PEcAn R packages</title>',
+  '</head>',
+  '<body>',
+  '<h1>PEcAn package documentation</h1>',
+  '<p>Function documentation and articles for each PEcAn package,',
+  '   generated from the package source using <a href="https://pkgdown.r-lib.org/" target="_blank">pkgdown</a> package.</p>',
```

Comment:
Nit: this sentence has too many copies of the word "package" in it, and this one's the easiest to remove 😉 
```suggestion
  '   generated from the package source using <a href="https://pkgdown.r-lib.org/" target="_blank">pkgdown</a>.</p>',
```